### PR TITLE
Add spec for module version

### PR DESCRIPTION
**Pull request by lonegunmanb**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## As part of this Pull Request I have

- [ ] Read the Contribution Guide and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [ ] Ensured PR tests are passing
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/specs/terraform/_index.md [hunk 1]
```diff
@@ -205,6 +226,8 @@ Module owners **MUST** use the below tooling for unit/linting/static/security an
 
 <br>
 
+#### ID: TFNFR6 - 
+
```
Comment (jtracey93) [inline docs/content/specs/terraform/_index.md:230]:
```suggestion
```

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Looks good just a bit of tidying up

**Review by jtracey93 — APPROVED** [review]
LGTM

**Comment by lonegunmanb** [comment]
I'm closing is pr in favour of #332

**Pull request closed without merging**